Sources/ContainerCommands/BuildCommand.swift

```diff
@@ -81,6 +81,12 @@ extension Application {
         )
         var memory: String = "2048MB"
 
+        @Option(
```

Comment:
Could you create a CLI test for a regular container and one for the builder?

For a regular container you could do something along the lines of this (using one of the alpine images we have on ghcr.io for testing):

```
% container run -it --rm alpine:latest sh -c "df -m / | tail -1 | tr -s ' ' | cut -d' ' -f2"
516064                           
```

For the builder you'd start the builder, `exec` the test command, and then stop the builder.